app/commands/beatmaps.py

```diff
@@ -99,3 +101,51 @@ async def add_beatmapset(context: Context):
             reference=context.message,
             mention_author=True
         )
+
+@app.session.commands.register(['modset'], roles=['BAT', 'Admin'])
+async def change_beatmapset_status(context: Context):
+    """<set_id> <status> - Modify a beatmapset status"""
+
+    if len(context.args) < 2 or not context.args[0].isnumeric():
+        await context.message.channel.send(
+            f'Invalid syntax: `!{context.command} <set_id> <status>`',
+            reference=context.message,
+            mention_author=True
+        )
+        return
+
+    set_id = int(context.args[0])
+
+    if context.args[1].isnumeric():
+        status = int(context.args[1])
+        if status not in range(-2, 5):
+            await context.message.channel.send(
+                f'Invalid status! Valid status: Ranked, Loved, Graveyard, numeric value',
+                reference=context.message,
+                mention_author=True
+            )
+            return
+    else:
+        statuses = {'ranked': 1, 'loved': 2, 'graveyard': -2}
```

Comment:
There are more statuses than this, which you can see here: https://github.com/osuTitanic/common/blob/61836e6f67789a020a672bec14c43fd3e9271d99/constants/status.py#L4C6-L4C6 (DatabaseStatus Enum)
Please add them as well!

Reply:
You could for example do
```python
statuses = {status.name:status.value for status in DatabaseStatus}
```